client/src/main/java/com/kritsit/casetracker/shared/domain/model/Evidence.java

```diff
@@ -10,7 +10,7 @@
     private String description;
     private File serverFile;
     private File localFile;
-    private BufferedImage image;
+    private transient BufferedImage image;
```

Comment:
This actually may get removed eventually anyway as Evidence is not always an image. Essentially this was intended to store the evidence (which to begin with was only images) locally the first time it was previewed so that it would not have to be fetched from the server every time (caching it essentially). However Evidence became more than just pictures and so the plan is simply to download them and save them locally temporarily while the application is running (in /tmp or something) and then access those.

So to answer your question - on the server as a file that is referenced in the DB and on the client temporarily once it is first accessed